regulations/templates/regulations/comment-confirm-fail.html

```diff
@@ -0,0 +1,20 @@
+<div>
+  Comment submission failed.
+  We're sorry, something went wrong with our system, and submitting your comment did not work.
+</div>
+
+<div>
+  <a href="{% url 'comment_review' doc_number %}">Return to your draft comment</a> to try submitting again.
+</div>
+
+<div>
+  If this problem keeps happening
+  If you've already tried submitting your comment more than once, please save a copy of your draft comment for safekeeping.
```

Comment:
This is copied from https://docs.google.com/document/d/1Rdgvue2PG0MioLez089Dlvlqln29F1RYWi3wpRV9trk/edit?ts=571f7b84.